### DISCUSSION TITLE
REST API: Return number of found rows?

### DISCUSSION THREAD

**Discussion by oguruma1218** [supabase/supabase#14404] [Questions]
I want to use a GET in a cURL call to search a table. I'd like to return the number of matching rows with the JSON. Is that possible to do natively?

**Comment by steve-chavez** [comment]
Use the `Prefer: count=exact` header.

https://postgrest.org/en/stable/references/api/tables_views.html#exact-count

**Comment by oguruma1218** [comment]
That seems to be what I need, however it doesn’t actually return anything new… When adding the Prefer: count=exact header, it still returns the JSON without any errors, but it doesn’t return the actual row count….

**Reply by GaryAustin1** [reply]
The results comes back in a header.
You might need to use rpc call if you can't get it out of the header.
